ykrt/src/compile/jitc_yk/codegen/x64/lsregalloc.rs

```diff
@@ -1156,7 +1159,7 @@ pub(crate) enum RegConstraint<R: Register> {
     Output,
     /// The result of this instruction will be stored in register `R`.
     OutputFromReg(R),
-    /// The register *r* will be clobbered.
+    /// The register *R* will be clobbered.
```

Comment:
I think "R" should be between backslashes?

Reply:
You are correct. Fixed in 3e88d2070a9e1821ddbffd3b34c0458f0ec97a76